lms/djangoapps/course_api/blocks/api.py

```diff
@@ -17,7 +17,9 @@ def get_blocks(
         requested_fields=None,
         block_counts=None,
         student_view_data=None,
-        return_type='dict'
+        return_type='dict',
+        block_types=None,
+        lti_url=None
```

Comment:
@lenacom @douglashall Would the LTI version ever affect these URLs? 

I'm still unclear about the differences between this and the endpoint documented here: https://github.com/edx/edx-platform/blob/a1c88bda5612f259b280776c68d06759ea2f75dc/lms/djangoapps/courseware/views.py#L1164-L1179
Maybe I could meet with you @douglashall to get cleared up?